go/tools/gazelle/resolve/resolve_external.go

```diff
@@ -117,6 +117,22 @@ func (r *externalResolver) lookupPrefix(importpath string) (string, error) {
 		subpaths = append(subpaths, prefix)
 	}
 
+	// gopkg.in is special, and might have either one or two levels of
+	// missing paths.
+	//
+	// https://github.com/bazelbuild/rules_go/issues/998
```

Comment:
Link to http://labix.org/gopkg.in instead of the bug.

Reply:
Done.